mscore/musescore.cpp

```diff
@@ -5783,7 +5783,7 @@ void MuseScore::realizeChordSymbols()
             return;
       if (!cs->selection().isList()) {
             QErrorMessage err;
-            err.showMessage("Invalid Selection. Cannot Realize Chord");
+            err.showMessage(tr("Invalid Selection. Cannot Realize Chord"));
```

Comment:
Hmm, I don't thing we should be using Title Case for those strings, sthese are normal sentences inside a n alert dialog, arent't they?

Reply:
(Sorry, I should have mentioned that earlier)

Reply:
Oh no worries, that makes sense though!